tests/test-verify-volume-cache.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: check cache options
+  block:
+
+    - name: Get information about the LV
+      command: "lvs --noheadings --nameprefixes --units=b --nosuffix --unquoted -o name,attr,cache_total_blocks,chunk_size,segtype
```

Comment:
This doesn't need to be double quoted.  I suggest using the YAML `>` string operator like this, so you can wrap the long line into multiple lines:
```
command: >
    lvs --noheadings --nameprefixes --units=b --nosuffix --unquoted
    -o name,attr,cache_total_blocks,chunk_size,segtype
    {{ storage_test_pool.name }}/{{ storage_test_volume.name }}
```
it's more ansible-lint/ansible-test/yamllint friendly